package.json

```diff
@@ -1,52 +1,54 @@
 {
-  "name": "homeassistant-sun-card",
+  "name": "lovelace-sun-card",
   "version": "1.0.0",
   "description": "Home assistant sun card based on Google weather design",
+  "type": "commonjs",
   "main": "dist/index.js",
   "scripts": {
-    "start": "rollup -c rollup.config.dev.js --watch",
+    "dev": "rollup -c rollup.config.dev.mjs --watch",
     "build": "rimraf dist && npm run lint && npm run rollup",
     "lint": "eslint src",
     "rollup": "rollup -c",
     "test": "jest"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/AitorDB/homeassistant-sun-card.git"
+    "url": "git+https://github.com/rejuvenate/lovelace-sun-card.git"
   },
-  "author": "AitorDB <aitor.db.mail@gmail.com>",
+  "author": "rejuvenate",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/AitorDB/homeassistant-sun-card/issues"
+    "url": "https://github.com/rejuvenate/lovelace-sun-card/issues"
   },
-  "homepage": "https://github.com/AitorDB/homeassistant-sun-card#readme",
+  "homepage": "https://github.com/rejuvenate/lovelace-sun-card#readme",
   "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-proposal-class-properties": "^7.16.0",
-    "@babel/plugin-proposal-decorators": "^7.16.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.16.0",
-    "@babel/plugin-syntax-decorators": "^7.16.0",
-    "@babel/preset-env": "^7.16.0",
-    "@babel/preset-typescript": "^7.16.0",
-    "@rollup/plugin-babel": "^5.3.0",
-    "@rollup/plugin-commonjs": "^21.0.1",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^13.0.6",
-    "@types/jest": "^27.0.2",
-    "@typescript-eslint/eslint-plugin": "^5.3.1",
-    "@typescript-eslint/parser": "^5.3.1",
-    "eslint": "^8.2.0",
-    "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "jest": "^27.3.1",
-    "rollup": "^2.60.0",
-    "rollup-plugin-serve": "^1.1.0",
-    "rollup-plugin-typescript2": "^0.30.0",
-    "ts-jest": "^27.0.7",
+    "@babel/core": "^7.12.3",
```

Comment:
Quite a few a of these got downgraded. I assume it was a side effect of starting this off the main branch before develop was merged so let's get the newer versions again (unless there was a legit reason to downgrade).